aeronautical/asterix/cat_034_catalog.ksy

```diff
@@ -0,0 +1,52 @@
+meta:
+  id: cat_034_catalog
+  license: GPL-3.0-only
+  endian: be
+  imports:
+#    - bit_map
+    - fixed
+    - explicit
+    - extended
+    - repetitive
+    - compound
+  ks-version: 0.9
```

Comment:
Normally, all top-level types would at bare minimum require the `title` to be set.